eng/verify-nupkgs.ps1

```diff
@@ -0,0 +1,76 @@
+[CmdletBinding()]
+Param(
+    [Parameter(Mandatory=$true)]
+    [System.String] $configuration
+)
+
+Add-Type -AssemblyName System.IO.Compression.FileSystem
+
+function Unzip {
+    param([string]$zipfile, [string]$outpath)
+
+    Write-Verbose "Unzipping '$zipfile' to '$outpath'."
+
+    [System.IO.Compression.ZipFile]::ExtractToDirectory($zipfile, $outpath)
+}
+
+function Test-NuGetPackages {
+    Write-Host "Starting Test-NuGetPackages."
+    $expectedNumOfFiles = @{
+        "MSTest.Internal.TestFx.Documentation"        = 58;
+        "MSTest.TestFramework"                        = 21;
+        "MSTest.TestAdapter"                          = 602;
+        "MSTest"                                      = 16;
+    }
+
+    $packageDirectory = Resolve-Path (Join-Path $PSScriptRoot "../artifacts/packages/$configuration")
+    $tmpDirectory = Resolve-Path (Join-Path $PSScriptRoot "../artifacts/tmp/$configuration")
+    $nugetPackages = Get-ChildItem -Filter "*.nupkg" $packageDirectory -Recurse -Exclude "*.symbols.nupkg" | ForEach-Object { $_.FullName }
+
+    Write-Host "Unzipping NuGet packages."
+    $unzipNugetPackageDirs = New-Object System.Collections.Generic.List[System.Object]
+    foreach ($nugetPackage in $nugetPackages) {
+        $unzipNugetPackageDir = $(Join-Path $tmpDirectory $(Get-Item $nugetPackage).BaseName)
```

Comment:
nit nit: the first subexpression $() is uneccessary, the second one can become just ( )